.github/CONTRIBUTING.md

```diff
@@ -85,7 +85,7 @@ named `phplist`, and the password is `batterystaple`, the command looks like
 this:
 
 ```bash
-mysql -u phplist_test --password=batterystaple phplist_test < vendor/phplist/core/Database/Schema.sql
+mysql -u phplist_test --password=batterystaple phplist_test < vendor/phplist/core/resources/resources/resources/Database/Schema.sql
```

Comment:
`/core/resources/resources/resources/Database/...` doesn't look right -- is it? Surely `/core/resources/Database/` ?

Reply:
Good catch! Fixed.